pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -154,6 +154,18 @@ function validateParams(vmParams) {
 
     if (vmParams.memorySize === 0) {
         validationFailed['memory'] = _("Memory must not be 0");
+    } else {
+        const osObj = vmParams.osInfoList.find(osElem => osElem.shortId == vmParams.os);
+        if (osObj &&
+            osObj['minimumResources']['ram'] &&
+            (convertToUnit(vmParams.memorySize, vmParams.memorySizeUnit, units.B) < osObj['minimumResources']['ram'])) {
+            validationFailed['memory'] = (
+                cockpit.format(
+                    _("The selected Operating System has minimum memory requirement of $0 $1"),
```

Comment:
Another nit-pick, but it's weird that other helpblock has sentence "The recommended memory for the selected Operating System is ...", while this sentence is written other way round, and at times both of these sentences are written next to each other. But perhaps @andreasn can tell it looks weird.

Reply:
Fixed.